<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Start Sorting?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttons_layout">
    <item>
     <layout class="QVBoxLayout" name="go_back_layout">
      <item>
       <widget class="QPushButton" name="go_back_button">
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Go Back...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="start_layout">
      <item>
       <widget class="QPushButton" name="start_button">
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start Sorting!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
